CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ include(FetchContent)
 FetchContent_Declare(
         fmt
         GIT_REPOSITORY https://github.com/fmtlib/fmt.git
-        GIT_TAG 8.1.1
+        GIT_TAG 10.0.0
```

Comment:
Does this affect the plan to eventually migrate to `std::format` and compatibility to the fmt version used in opencmw? (As we've been bitten by fmt version mismatches before)

Reply:
Shouldn't ... most of the issues I found were related to sloppy/missing/implicit formatted conversion... having changed to these stricter constraints and moving to std::format should make this rather easier...